account/accountinfrastructure/accountmemory/workspace.go

```
@@ -46,6 +50,34 @@
 	}), rerror.ErrNotFound)
 }
 
+func (r *Workspace) FindByUserWithPagination(ctx context.Context, id user.ID, pagination *usecasex.Pagination) (workspace.List, *usecasex.PageInfo, error) {
+	if r.err != nil {
+		return nil, nil, r.err
+	}
+
+	if pagination == nil {
+		pagination = &usecasex.Pagination{}
```

golangci-lint (1.64.8)

ineffectual assignment to pagination

(ineffassign)